conf/application.conf

```diff
@@ -116,3 +116,262 @@ mongodb {
   uri = "mongodb://localhost:27017/merchandise-in-baggage-frontend"
 }
 
+autocomplete {
```

Comment:
Think this might as well be in one, we're not likely to want different countries in different environments. It would be less error prone and easier to maintain in code too.